app/src/main/java/org/fossasia/openevent/app/qrscan/ScanQRActivity.java

```diff
@@ -251,4 +246,14 @@ public void startScan() {
     public void stopScan() {
         cameraSource.stop();
     }
+
+    public void showToggleDialog(long attendeeId) {
+        BottomSheetDialogFragment bottomSheetDialogFragment = AttendeeCheckInFragment.newInstance(attendeeId);
+        bottomSheetDialogFragment.show(getSupportFragmentManager(), bottomSheetDialogFragment.getTag());
+    }
+
+    @Override
+    public void onDialogDismissed() {
+        presenter.allowScan(true);
```

Comment:
>simply stop scanning when it is shown and resume when dismissed

couldn't figure out how to stop and resume scanning. will need your help. @iamareebjamal 
This is just work around :smile: